docker/jenkins-test.sh

```diff
@@ -46,14 +46,14 @@ function run_tests {
 
     # create database and user once
     echo "creating user and database"
-    docker-compose -f $COMPOSE_FILE_LOC -p $COMPOSE_PROJECT_NAME run --rm $TEST_CONTAINER_NAME \
+    docker-compose -f $COMPOSE_FILE_LOC -p $COMPOSE_PROJECT_NAME run --rm $TEST_CONTAINER_NAME --name $TEST_CONTAINER_REF\
```

Comment:
I don't think we need `--name` here - we're already using `--rm`. This container only runs init_db and then quits and removes itself.